### DISCUSSION TITLE
求助：官方提到的 src-tauri 目录下 pake.json 文件，是手动创建吗？

### DISCUSSION THREAD

**Discussion by liumu927** [tw93/Pake#823] [Q&A]
**前提：** 我有一个本地项目，全局安装了`pake-cli` 后，可以通过 `pake [url]` 进行打包和安装，能够运行。

**需求：** 需要进行“高级用法”定制开发

**问题1：** 官方md文档提到——需要修改`src-tauri/src/inject/style.js`或者`src-tauri/pake.json`中的内容，我想知道使用`pake`的话也是需要通过`Tauri`中的初始化`npm run tauri init`进行生成是吗？
`npm run tauri init`后的目录结构如下：
![image](https://github.com/user-attachments/assets/d92cd6cc-e9ca-4ad4-81c2-1f847c341226)
但是这种方法生成的`src-tauri`目录 **并不包含pake的相关内容，如pake.json。** 我并未在Pake官方文档中找到这部分应该如何创建。
官方给出的`src-tauri`目录结构应该是这样：
![image](https://github.com/user-attachments/assets/c155bf50-9d0d-45ff-8922-87cf1aa9beb6)

**问题2**：是否支持如`Tauri`中`npm run tauri dev`应用程序的开发版本？便于在开发过程中进行调试

**Comment by Tlntin** [comment]
`我想知道使用pake的话也是需要通过Tauri中的初始化npm run tauri init进行生成是吗？` 不需要，用现在的项目就行
`问题2` 支持，可以看package.json里面有命令。

**Reply by liumu927** [reply]
`不需要，用现在的项目就行` ，若不使用npm run tauri init初始化，我的项目中并没有src-tauri目录及目录下的pake的相关内容，您的意思是直接将Pake中的`src-tauri目录`直接复制到我自己的项目中吗？
![image](https://github.com/user-attachments/assets/0e484ed3-cee7-4818-8676-3e34774ece95)

**Reply by Tlntin** [reply]
可以试试直接复制，不过可能要魔改一些东西。你可以另外起一个项目看看默认模板有啥东西。

**Reply by liumu927** [reply]
好的我试试 谢谢
